README.md

```diff
@@ -12,16 +16,38 @@ This GitHub Action will query Jira using JQL provided as an input, and will tran
   - [Code of Conduct](#code-of-conduct)
   - [License](#license)
-      
+
 
 ## Inputs
-| Parameter         | Is Required | Description                                                                                                                                                           |
-| ----------------- | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `domain-name`     | true        | The domain name for Jira.                                                                                                                                             |
-| `jql-query`       | true        | The JQL query to use to find tickets that will be transitioned.                                                                                                       |
-| `transition-name` | true        | The name of the transition to perform. Examples might include Open, In Progress, Deployed, etc.                                                                       |
-| `jira-username`   | false       | The username to login to Jira with in order to perform the transition. Will be ignored if not set.                                                                    |
-| `jira-password`   | false       | The password to login to Jira with in order to perform the transition. Must be set if jira-username is set. If set when jira-username is not set, it will be ignored. |
+
+| Parameter                          | Is Required    | Description                                                                                                                                                                                                                      |
+|------------------------------------|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `domain-name`                      | true           | The domain name for Jira.                                                                                                                                                                                                        |
+| `jql-query`                        | conditionally* | The JQL query to use to find tickets that will be transitioned. A max of 20 issues can be transitioned.                                                                                                                          |
+| `issues`                           | conditionally* | Comma delimited list of issues to transition. Use `im-open/get-workitems-action` to identify list of issues for a PR or deployment.                                                                                              |
```

Comment:
The project uses issues, tickets and tasks  terms to reference a Jira Issue.

Would it be possible to consolidate on one term?  Notice the repo name as well is using task as a term.

Work Item is what the `im-open/get-work-items-action` uses.  But it is more generic.

Reply:
That's a good point. This action was intended to transition deployment tasks which is why it has the name it does. Maybe that's not the best term to use any longer though. It seems like the Jira documentation leans towards issues, so maybe that's the way to go.